.github/workflows/e2e_other_tests.yml

```diff
@@ -74,12 +74,24 @@ jobs:
         with:
           name: protractor-video
           path: /home/runner/work/oppia/protractor-video
+      - name: Uploading webdriverio-video as Artifacts
+        if: ${{ steps.check-risk.outputs.is-low-risk != 0 && always() }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: webdriverio-video
+          path: /home/runner/work/oppia/webdriverio-video
       - name: Uploading protractor screenshots as Artifacts
         if: ${{ steps.check-risk.outputs.is-low-risk != 0 && always() }}
         uses: actions/upload-artifact@v2
         with:
           name: protractor-screenshots
           path: /home/runner/work/oppia/protractor-screenshots
+      - name: Uploading webdriverio screenshots as Artifacts
```

Comment:
If we have webdriverio screenshots, should we stop adding protractor screenshots as artifacts?

Reply:
I have not migrated all the test that is part of e2e_other_tests.yml which means there are still protractor test that will generate screenshots as articfacts. Once all the test of e2e_other_tests.yml are migrated to webdriverio we can remove it.